README.md

```diff
@@ -3,7 +3,7 @@
 TODO (https://github.com/UBC-Thunderbots/Software/issues/1478): Uncomment this once Appveyor is up and running again
 [![Build status](https://ci.appveyor.com/api/projects/status/a7106hhbcal8kqi8?svg=true)](https://ci.appveyor.com/project/UBCThunderbots/software)
 -->
-[![Build Status](https://travis-ci.org/UBC-Thunderbots/Software.svg?branch=master)](https://travis-ci.org/github/UBC-Thunderbots/Software/builds)
+![Tbots CI](https://github.com/UBC-Thunderbots/Software/workflows/Tbots%20CI/badge.svg)
```

Comment:
clicking on the icon should link to the github actions ci for master

Reply:
fixed